Assets/Scripts/ssjAug2018/Items/Mail.cs

```diff
@@ -89,6 +90,8 @@ public void Throw(Player owner, Vector3 origin, Vector3 velocity)
             _rigidbody.velocity = velocity;
 
             _despawnTimer.Start(ItemManager.Instance.ItemData.MailDespawnSeconds, Miss);
+
+            AudioManager.Instance.PlayOneShot(ItemManager.Instance.ItemData.ThrowMailAudio);
```

Comment:
Move this to the CmdThrowMail in the Player class and only play if(IsLocalActor)